### ISSUE TITLE
assigned ticket threshold for warning is static

### ISSUE THREAD

**Issue by ntozier** [osTicket/osTicket-1.7#552]
/scp/tickets.php
circa line 495 though 497

if($stats['assigned']) {
    if(!$ost->getWarning() && $stats['assigned']>10)
        $ost->setWarning($stats['assigned'].' tickets assigned to you! Do something about it!');

10 should be admin configurable say in admin panel -> settings -> Staff Excessive Tickets [#]

**Comment by greezybacon** [comment]
I agree. We should do this. The new config system makes this extremely easy
